hal/riscv64/generic/Makefile

```diff
@@ -10,13 +10,14 @@ CFLAGS += -DVADDR_KERNEL_INIT=$(VADDR_KERNEL_INIT)
 
 GCCLIB := $(shell $(CC) $(CFLAGS) -print-libgcc-file-name)
 
-PLO_COMMANDS = alias app call console copy dump echo go help kernel map mem phfs reboot script wait
+PLO_COMMANDS ?= alias app call console copy dump echo go help kernel map mem phfs reboot script wait
 
-include devices/ram-storage/Makefile
+# TODO: Consider how conflicting devices should be handled
```

Comment:
this particular case probably should be handled by specifying correct `PLO_DEVICES` in project file ;)

Reply:
I am not sure. 
If I correctly understand code, when I put both devices in ALLDEVICES and DEVICES will not fix that, then both will be compiled. So constructor order will determine which driver will be used (the last registered) without any warning.

Reply:
I was thinking about about users specifying correct `PLO_DEVICES`. Right now the constructors are being sorted by priority (which is not set explicitly, so I assume they have the same priority), I don't know if the current constructors are sorted the same in each compilation. 

AFAIK you use device by MAJOR.MINOR in plo script so having 2 drivers for the same device is not an error per se (but not a perfect mechanism for specifying a driver for device). Also - the bootloader should be as small as possible, so there shouldn't be multiple drivers for one device in the final binary.

Regarding guarding against invalid `PLO_DEVICES` specification by user - we might want to prevent initializing multiple drivers accessing the same peripheral (either at runtime by directly assigning major/minor and failing `init()` if not as expected? or at compile time (eg. exporting symbol with the device name (eg. `extern const int __uart_device`) so linking would fail). It's a separate topic than this PR so I would not stress about it here.

Reply:
@nalajcie perhaps this could be fixed by introducing changes analogous to changes in https://github.com/phoenix-rtos/plo/commit/291dd5e02fd908d1c77b067af04c9cb0fbc86999? This would make minor number deterministic.

Reply:
@agkaminski this is to be done in https://github.com/phoenix-rtos/plo/pull/315 (in progress)

Reply:
So do you want me to:
1. Add commit in plhoenix-rtos-project with proper configuration and fiill ALLDEVICES with (heh) all devices or
2. Leave the current state or
3. Add some more warning?

Reply:
Preferably - (1). If you don't have the incentive, let's (2) and make it better with PR #315.